features/support/save_to_dir_embedding_formatter.rb

```diff
@@ -34,8 +34,17 @@ def initialize(config)
 
     # @note #embed has been replaced by #attach in Cucumber 5.x
     def attach(src, media_type)
-      if media_type != 'text/x.cucumber.log+plain'
-        embed(src, media_type, "test case execution attachment")
+      if media_type == 'text/x.cucumber.log+plain'
+        puts src
+      elsif media_type == 'image/png;base64'
+        filename = Time.now.iso8601 + '-screenshot.png'
+        embed(src, media_type, "#{filename}")
+      elsif media_type == 'text/html;base64'
+        filename = Time.now.iso8601 + '.html'
+        embed(src, media_type, "#{filename}")
```

Comment:
maybe we don't need to save html file

Reply:
It comes from https://github.com/openshift/verification-tests/pull/2025/files#diff-4e2de23aa38bbb77f3e035cf102da99897869c1bae03520d60ae31664cd7819dL667
Just keep it to work as the old fashion.

If you persist it's not needed, I can remove them.

Reply:
I'm ok to keep it and we can still remove it when we think the large artifact is a problem